src/foam/comics/v2/DAOSummaryView.js

```diff
@@ -107,8 +107,12 @@ foam.CLASS({
   actions: [
     {
       name: 'edit',
-      isAvailable: function(config$editEnabled) {
-        return config$editEnabled;
+      isAvailable: function(config) {
+        try {
+          return ! config.update || ! config.update.f  ? true : config.update.f();
```

Comment:
Why is this needed? Why can't you just always call config.update.f()?

Reply:
yes, this was before each method was changed to a Predicate.  And the issue is no default value of a TRUE predicate. 
return config.update.f()

Reply:
fixed https://github.com/foam-framework/foam2/pull/3234